schedule/yast/remote_ssh/remote_ssh_controller_sle15.yaml

```diff
@@ -0,0 +1,28 @@
+---
+name: remote_ssh_controller
+description: >
+  Multi-machine test using one VM to trigger installation over ssh using ftp
+  mirror, second VM connects over ssh and conducts the installation.
+  Installation is validated by successful boot and that YaST does not report
+  any issue. Installs remote server (parallel job) with ssh.
+schedule:
```

Comment:
There are multiple variables for this test suite and its companion below that could be added (some here, some in job group yaml): REMOTE_CONTROLLER, SUPPORT_SERVER, SUPPORT_SERVER_ROLES, VIDEOMODE, etc. I think this is the only test suite that also run in openSUSE for this batch. Is there some reason why you didn't add them? (maybe I'm missing something that you know).

Reply:
The variables are set in Test Suite settings. As we agreed on meeting, it is not a part of the task to reduce amount of Test Suite settings, I'm just moving the tests to yaml schedule.

Usually, I'm also trying to define which variables might be left in yaml schedule or moved to Job Group, but in this case I'm not too familiar with the scenario and I'm not sure which variables are used as test data and which ones are used to setup the environment.

Reply:
agreed offline that optimizing test suites or variables is an optional choice.